Bigtable/src/Table.php

```diff
@@ -0,0 +1,352 @@
+<?php
+/**
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Google\Cloud\Bigtable;
+
+use Google\Cloud\Bigtable\Admin\V2\BigtableTableAdminClient as TableAdminClient;
+use Google\Cloud\Bigtable\Connection\ConnectionInterface;
+use Google\Cloud\Core\Exception\NotFoundException;
+use Google\Cloud\Core\Iterator\ItemIterator;
+use Google\Cloud\Core\LongRunning\LongRunningConnectionInterface;
+use Google\Cloud\Core\LongRunning\LongRunningOperation;
+use Google\Cloud\Core\LongRunning\LROTrait;
+
+/**
+ * Represents a Cloud Bigtable table
+ *
+ * Example:
+ * ```
+ * $table = $instance->table('my-table');
```

Comment:
Class examples should start with creating a client. So you'd need to do something like this:

```php
use Google\Cloud\Bigtable\BigtableClient;

$bigtable = new BigtableClient();
$instance = $bigtable->instance('my-instance');

$table = $instance->table('my-table');
```

I think of it as the method examples follow from the class example. If I copy and paste the class example, followed by copying and pasting a method example into an empty file, I should have a 100% valid code snippet for any method. :)